src/helpers/review.js

```diff
@@ -0,0 +1,30 @@
+
+import fetch from "node-fetch";
+
+const API = "https://plankton-app-xhkom.ondigitalocean.app/api/reviews";
+
+export async function sendReview(movieId, data) {
+
+  const body = {
+    "data": {
```

Comment:
This could just be `data` (not `"data"`) to be consistent with the following lines.